src/components/additem/TextInput.tsx

```diff
@@ -1,11 +1,22 @@
+import { ChangeEvent } from "react";
 import "./TextInput.css";
 
-function TextInput({ isTextArea, name, label, placeholder, onChange }) {
+interface Props {
```

Comment:
그냥 `Props` 보다 어떤 컴포넌트의 `Props`인지 이름을 명확하게 주면 좋을 것 같습니다 !

```suggestion
interface TextInputProps {
```